backend/catalogue/viewsets.py

```diff
@@ -3,8 +3,8 @@
 # Released under the Affero General Public Licence (AGPL) v3.  See LICENSE file for details.
 from django.db.models import Value
 from django.db.models.functions import Coalesce
-from catalogue.models import Organisation, HabitatDescriptor, BaseLayer, Layer, LayerGroup, LayerGroupPriority, DataClassification, BaseLayerGroup, SaveState
```

Comment:
These are getting quite long -- let's clean up while we're at it ("import ..as models", then refer as models.BaseLayer, etc)

Reply:
Good idea

Reply:
Imports cleaned-up